### PR TITLE
Turkish Fixes

### PR DESCRIPTION
**Pull request by DayibBaba**

### CODE REVIEW

File: tr_TR.json [hunk 1]
```diff
@@ -427,10 +427,10 @@
   "DEBUG_WINDOW": "Hata ayıklama",
   "DEFAULT": "Varsayılan",
   "DEFAULT_CMD_PERMISSIONS": "Varsayılan Komut İzinleri",
-  "DELETE": "Silmek",
+  "DELETE": "Silm",
   "DELETE_TRAIN": "Treni Sil",
   "DELETE_TRAIN_DESC": "Trenin içindeyken treni silemezsiniz.",
-  "DERAIL_TRAIN": "Raydan Çıkan Tren",
+  "DERAIL_TRAIN": "Raydan Çıkan",
```
Comment (ShinyWasabi) [inline tr_TR.json:433]: "Treni Raydan Çıkar".

File: tr_TR.json [hunk 2]
```diff
@@ -494,11 +494,11 @@
   "FIRE": "Ateş",
   "FLIGHT_SCHOOL": "Uçuş okulu",
   "FOLDER": "Dosya",
-  "FORCE": "Güç",
+  "FORCE": "Zorla",
   "FORCE_HOST": "Ana Bilgisayarı Zorla",
   "FORCE_INTO_MISSION": "Göreve Zorla Girin",
   "FORCE_INTO_MISSION_DESC": "Oyuncuyu silah kaçakçılığı görevine zorlar",
-  "FORCE_MISSION": "Kuvvet Misyonu",
+  "FORCE_MISSION": "Misyona Zorla",
```
Comment (ShinyWasabi) [inline tr_TR.json:501]: "Göreve Zorla".

File: tr_TR.json [hunk 3]
```diff
@@ -591,13 +591,13 @@
   "GUI_TAB_PROTECTION_SETTINGS": "Koruma",
   "GUI_TAB_PROXY_SETTINGS": "Vekil sunucu Ayarları",
   "GUI_TAB_REACTION_SETTINGS": "Tepkiler",
-  "GUI_TAB_SELF": "öz",
+  "GUI_TAB_SELF": "Kendim",
   "GUI_TAB_SESSION_BROWSER": "Oturum Tarayıcısı",
   "GUI_TAB_SETTINGS": "Ayarlar",
   "GUI_TAB_SPAWN_PED": "Yumurtlama Ped",
-  "GUI_TAB_SPAWN_VEHICLE": "Doğma Aracı",
+  "GUI_TAB_SPAWN_VEHICLE": "Arac Spawnlama",
```
Comment (ShinyWasabi) [inline tr_TR.json:598]: "Araç Spawnlama".

File: tr_TR.json [hunk 4]
```diff
@@ -620,7 +620,7 @@
   "HANDLING_DOWNSHIFT_MULTIPLIER": "Vites Küçültme Çarpanı",
   "HANDLING_DRIVE_BIAS_FRONT": "Sürüş Önyargısı (1,0 = ön, 0,0 = arka, 0,5 = dengeli 4WD)",
   "HANDLING_DRIVE_BIAS_REAR": "Sürüş Önyargısı (1,0 = arka, 0,0 = ön, 0,5 = dengeli 4WD)",
-  "HANDLING_ENTER_VEHICLE": "Lütfen bir araç giriniz.",
+  "HANDLING_ENTER_VEHICLE": "Lütfen bir araça giriniz.",
```
Comment (ShinyWasabi) [inline tr_TR.json:623]: "Lütfen bir araca giriniz."

File: tr_TR.json [hunk 5]
```diff
@@ -728,8 +728,8 @@
   "KEEP_VEHICLE_FIXED": "Aracı Tamir Edin",
   "KEEP_VEHICLE_FIXED_DESC": "Aracınızı aşınma ve yıpranmadan korur",
   "KEEP_VEHICLE_REPAIRED": "Aracı Onarılmış Tut açıksa, aracın sağlığını değiştirmek için devre dışı bırakın.",
-  "KICK": "Tekme atmak",
-  "KICKED": "tekmelendi",
+  "KICK": "Tekmele",
```
Comment (ShinyWasabi) [inline tr_TR.json:731]: I think "Oyundan At" or "Sunucudan At" would be a better term for kicking someone.

File: tr_TR.json [hunk 6]
```diff
@@ -728,8 +728,8 @@
...
+  "KICKED": "Tekmelendi",
```
Comment (ShinyWasabi) [inline tr_TR.json:732]: "Oyundan Atıldı" or "Sunucudan Atıldı".

**Comment by ShinyWasabi** [comment]
> @ShinyWasabi Are you familliar with github cli? If so, I invite you and the other user who had tried to contribute before, to help me with this as im not rich with time at present.

Not really, but I think I can figure it out.

**Comment by DayibBaba** [comment]
> > @ShinyWasabi Are you familliar with github cli? If so, I invite you and the other user who had tried to contribute before, to help me with this as im not rich with time at present.
> 
> Not really, but I think I can figure it out.

I can help set it up, just holler.

**Comment by DayibBaba** [comment]
> > > > @ShinyWasabi Are you familliar with github cli? If so, I invite you and the other user who had tried to contribute before, to help me with this as im not rich with time at present.
> > > 
> > > 
> > > Not really, but I think I can figure it out.
> > 
> > 
> > I can help set it up, just holler.
> 
> He shouldn't have to learn to be a Linux dork to contribute. I think if you just give him push privileges to your repo, he can just make a commit with his fixes instead of trying to get him to pretend it's 1975.

May I remind you, that you are way older than me.

**Comment by gir489returns** [comment]
> > > > > @ShinyWasabi Are you familliar with github cli? If so, I invite you and the other user who had tried to contribute before, to help me with this as im not rich with time at present.
> > > > 
> > > > 
> > > > Not really, but I think I can figure it out.
> > > 
> > > 
> > > I can help set it up, just holler.
> > 
> > 
> > He shouldn't have to learn to be a Linux dork to contribute. I think if you just give him push privileges to your repo, he can just make a commit with his fixes instead of trying to get him to pretend it's 1975.
> 
> May I remind you, that you are way older than me.

Which just makes it even more sad that you want to live in the past.

**Pull request merged**